src/test/java/dk/kb/image/FacadeTest.java

```diff
@@ -0,0 +1,101 @@
+package dk.kb.image;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.imageio.ImageIO;
+import dk.kb.util.Resolver;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FacadeTest {
+    private Logger log = LoggerFactory.getLogger(this.toString());
+
+
+    // method to validate that image is greyscale
+    public static boolean isGreyscale(BufferedImage image){
```

Comment:
This method is a helper, along with `canCountColors` so they should be _after_ the primary methods (the ones annotated with `@Test`).

Reply:
Has been moved